tests/test_other.py

```diff
@@ -7970,7 +7970,7 @@ def run(*args):
       run(['-Os'],  0, [], [],          85,  0,  2,  2) # noqa
       run(['-Oz'],  0, [], [],          54,  0,  1,  1) # noqa
     else:
-      run([],      20, ['abort'], ['waka'], 22712, 20, 14, 27) # noqa
+      run([],      21, ['abort'], ['waka'], 22712, 22, 15, 30) # noqa
```

Comment:
it's very odd this would change, as the program doesn't use syscalls. do you know what's going on here?

Reply:
Yes I thought that was odd too, let me look a bit more closely

Reply:
The extra exports/imports are due to new dyncalls using 64bit ints.